ydb/public/api/protos/ydb_topic.proto

```diff
@@ -796,24 +811,89 @@ message AlterConsumer {
     map<string, string> alter_attributes = 6;
 }
 
+enum AutoScalingStrategy {
+    // Auto scaling disabled
+    DISABLED = 0;
+    // The auto scaling may only increase the number of partitions
+    SCALE_UP = 1;
+    // The auto scaling may increase and decrease the number of partitions
+    SCALE_UP_AND_DOWN = 2;
+}
+
+message AutoScalingPartitionWriteSpeedStrategy {
```

Comment:
What about place `AutoScalingPartitionWriteSpeedStrategy` after or into `AutoScalingSettings`?

Reply:
it will move details after main message - cosmetic and understandable only, no tech reason.